templates/tpus.libsonnet

```diff
@@ -22,6 +22,7 @@ local base = import 'base.libsonnet';
     type: 'tpu',
     version: error 'Must override `version`',
     size: error 'Must override `size`',
+    numCores: if tpu.version <= 3 then 8 else 4,
     replicas: tpu.size / 8,  // Each TPU replica has 8 cores
```

Comment:
do we also need to change this?

`replicas: tpu.size / self.numCores`

Reply:
This calculation should still be correct -- `replicas` essentially means hosts in this context.